### PR TITLE
Liking posts

### PR DESCRIPTION
**Pull request by karimabuseer**

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
+  end
 
- def find_post
-   @post = Post.find(params[:post_id])
+  def find_like
+    @like = @post.likes.find(params[:id])
  end
 
+ def destroy
+  if (already_liked?)
```
Comment (YoFirmy) [inline app/controllers/likes_controller.rb]: do we need the if statement?

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -23,8 +23,21 @@
     <% post.images.each do |image| %>
     <img src="<%=(url_for(image))%>" class="post_image post-images"/>
     <% end %>
+
+
```
Comment (YoFirmy) [inline app/views/posts/index.html.erb]: can we get rid of multiple empty lines

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -23,8 +23,21 @@
...
+<% if pre_like %>
+  <%= button_to 'Unlike', post_like_path(post, pre_like), method: :delete, id: "ID#{post.id}", class:'btn btn-primary' %>
+<% else %>
+ <%= button_to 'Like', post_likes_path(post.id), method: :post ,id: "ID#{post.id}", class:'btn btn-primary like-button'%> 
+<% end %>
+    <%
+=begin%>
+ <%= button_to 'Like', post_likes_path(post.id), method: :post ,id: "ID#{post.id}", class:'btn btn-primary like-button'%> 
+<%
+=end%>
```
Comment (YoFirmy) [inline app/views/posts/index.html.erb]: can we fix indentation to make more readable and also get rid of commented out code

**Review by YoFirmy — CHANGES_REQUESTED** [review]
I'm not seeing any changes to the javascript files?... surely there are some?

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (karimabuseer) [reply app/controllers/likes_controller.rb]: Removed. Tightened the controller.

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (karimabuseer) [reply app/views/posts/index.html.erb]: Done, and added proper indentation to several lines.

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (karimabuseer) [reply app/views/posts/index.html.erb]: Done

File: app/assets/javascripts/likes.js [hunk 4]
```diff
@@ -1,25 +1,79 @@
 document.addEventListener('DOMContentLoaded', () => {
   const posts = document.querySelectorAll('.post');
-  posts.forEach((post) => {
-    post.querySelector('.like-button').addEventListener('click', (event) => { 
+
+  function loadComments() { posts.forEach((post) => {
```
Comment (YoFirmy) [inline app/assets/javascripts/likes.js:4]: fix indentation

**Review by YoFirmy — CHANGES_REQUESTED** [review]

**Pull request merged**